src/modules/blocks/event-datetime/subblocks/dashboard/container.js

```diff
@@ -24,84 +25,31 @@ import EventDateTimeDashboard from './template';
  */
 
 const onSelectDay = ( stateProps, dispatchProps ) => ( { from, to } ) => {
```

Comment:
we could remove `stateProps` from all these functions as we aren't using them anymore.